src/manage_sql.c

```diff
@@ -15631,6 +15631,11 @@ update_nvti_cache ()
                  " FROM nvts"
                  " LEFT OUTER JOIN vt_refs ON nvts.oid = vt_refs.vt_oid;");
 
+  init_iterator (&prefs,
+                 "SELECT pref_id, pref_nvt, pref_name, value"
+                 " FROM nvt_preferences"
+                 " WHERE NOT (pref_type = 'entry' AND pref_name = 'Timeout')");
```

Comment:
Shouldn't this use lower case for the pref_name?
```suggestion
                 " WHERE NOT (pref_type = 'entry' AND pref_name = 'timeout')");
```

Reply:
Yes, good one.

Reply:
Done in [1439e4f](https://github.com/greenbone/gvmd/pull/2096/commits/1439e4ffb298f8a1cc50b0d032672bb013bb8da0)